frontend/src/components/smart/CharacterList.vue

```diff
@@ -165,15 +155,24 @@ export default {
 
       this.$emit('character-filters-changed');
     },
+
+    priceSortText(type) {
```

Comment:
this could be done a bit better:

```
const sorts = [
 { name: 'Any', dir: 0 },
 { name: 'Price: Low -> High', dir: 1 },
 { name: 'Price: High -> Low', dir: -1 },
]
```
then use that. then we can add more sorts later without so much boilerplate.